src/main/java/com/googleinterns/zoomtube/utils/LectureUtil.java

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.googleinterns.zoomtube.utils;
+
+import com.googleinterns.zoomtube.data.Lecture;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Key;
+
+/** Provides methods to create Lecture Entities and Lectures. */
+public class LectureUtil {
```

Comment:
Please make final and make the constructor private to enforce that this is a Utils class. It will help to make sure that this class cannot be subclassed and that it non-instatiable.